oat-sa/tao-test-runner-qti-fe
# [TCA-590] Adjust the Keyboard Navigation overlay with correct Information

Related to : https://oat-sa.atlassian.net/browse/TCA-590
 
Refactor jumplinks shortcuts to make it accessibility friendly
  
#### How to test
 
- prepare an instance of TAO with taoAct extension
- prepare a delivery with enabled accessibility feature
- execute delivery as a test taker
- activate shortcuts jump link
- check AC from related ticket
  
#### Dependencies

Companion PR :
 - [ ] https://github.com/oat-sa/extension-tao-act/pull/1386
 - [ ] https://github.com/oat-sa/extension-tao-testqti/pull/1809
